CODEOWNERS

```diff
@@ -282,6 +282,8 @@ extensions/filters/http/oauth2 @derekargueta @mattklein123
 /*/extensions/filters/http/ip_tagging @alyssawilk @JuniorHsu
 # Header to metadata
 /*/extensions/filters/http/header_to_metadata @zuercher @JuniorHsu
+# Json to metadata
+/*/extensions/filters/http/json_to_metadata @JuniorHsu @UNOWNED
```

Comment:
A new extension needs a maintainer sponsor to be added to the main repository. Please find a maintainer who will take ownership together with you. Otherwise you can add it to the contrib directory.

Reply:
updated